tutorials/polkadot-sdk/parachains/build-custom-pallet/build-pallet.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Build the Pallet
```

Comment:
For SEO purposes, this title could be improved because it doesn't provide much context to users who are browsing on google, etc.

Reply:
This is just a placeholder, we will remove this section when restructuring the tutorial sections